civihr_employee_portal/js/reports.js

```diff
@@ -203,10 +205,83 @@
 
         $(this).find('.pvtCheckContainer p').addClass('chr_custom-checkbox');
       }
+
+      // Altering Employee length of service data to show number of days into
+      // String format of Year Months and days of length of service.
+      // Finding the field called Employee length of service.
+      if($(this).find('h4').text().indexOf("Employee length of service (") >= 0 ) {
+         // Iterating through all the filter values.
+        $(this).find('p.chr_custom-checkbox').each(function(i){
+          // Skip the Select All checkbox from filters.
+          if (!$(this).hasClass("pvtFilterSelectAllWrap")){
+            // Fetching the Employee length of service from filter values.
+            var employee_length_service = $(this).find('span:first').text();
+            // Replacing with format string calling function as parameter.
+            $(this).find('span:first').text(that.formatLengthsOfService(employee_length_service));
+          }
+        });
+      }
     });
   };
 
   /**
+   * Processes results in people report view to format length of service for each
+   * contact in a human readable form.
+   *
+   * @param employee_length_service
+   *
+   * @returns string
+   *
+   */
+  HRReport.prototype.formatLengthsOfService = function (employee_length_service) {
+
+    // Getting current date.
+    var date_current = new Date();
+    // Calculating date difference from length of service for Employee using current date.
+    var utime_target = date_current.getTime() + employee_length_service * 86400 * 1000;
```

Comment:
I've changed 86400 to (24*60*60) its basically adding time to date. 1000 is milliseconds and utime is basically a variable containing unix timestamp.